etl/steps/data/garden/antibiotics/2024-10-23/animuse_year.meta.yml

```diff
@@ -0,0 +1,38 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Global Health
+        - Antibiotics
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
+
+
+tables:
+  animuse_year:
+    variables:
+      mg_per_kg_non_adjusted:
+        title: Antimicrobial quantities adjusted for animal biomass (non-adjusted for reporting coverage)
```

Comment:
you could improve the metadata here too based on some of my previous suggestions :)